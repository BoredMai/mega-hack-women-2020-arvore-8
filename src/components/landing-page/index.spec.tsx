import { shallow, ShallowWrapper } from 'enzyme';
import React from 'react';

import LandingPage from '.';

describe('<LandingPage>', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() => {
    wrapper = shallow(<LandingPage />) as ShallowWrapper;
  });

  it('renders properly', () => {
    expect(wrapper.getElement()).toMatchSnapshot();
  });
});
