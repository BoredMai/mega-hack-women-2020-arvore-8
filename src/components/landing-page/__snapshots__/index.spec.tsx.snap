// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingPage> renders properly 1`] = `
<div
  className="content-wrapper"
>
  <h1>
    Colha uma História
  </h1>
  <p>
    Clarice, seja a autora do seu próprio conto e compartilhe com todos suas aventuras!
  </p>
  <button>
    <Link
      to="/biblioteca"
    >
      Vamos lá
    </Link>
  </button>
</div>
`;
